app/clouddeploy.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
   name: demo-test
 description: test cluster
 gke:
-  cluster: projects/cloudy-test1/locations/us-west1-c/clusters/demo-test
+  cluster: projects/cicd-demo-375021/locations/us-west1-c/clusters/demo-test
```

Comment:
We should parameterize this vs changing with each PR

Reply:
Swapped it to PROJECT_ID as a placeholder